src/components/pages/TeachersPage/index.tsx

```diff
@@ -156,6 +365,29 @@ export default function TeachersPage({ classroomName }: ClassroomProps) {
           {showDisplayedChat()}
         </Grid>
       </Grid>
+      <Grid container spacing={2} mt={2} pb={2}>
+        <Grid item xs={12}>
+          <Typography variant='h5' sx={{ color: 'white' }}>
+            Click to view full chat
```

Comment:
Love the idea of clicking to view the full chat.  Let's change text to to be more descriptive.  "Click any conversation below to display full chat above".